app/views/veterans/profile/_announcements.html.erb

```diff
@@ -0,0 +1,3 @@
+<% if @announcements.nil? %>
+  <center><i>No announcements</i></center>
```

Comment:
Per [i](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/i#Notes)'s documentation, use newer tags or CSS to italicize the text.